locales/de/index.yml

```diff
@@ -1690,6 +1717,8 @@ wallet:
       WAITING_CONFIRMATION_EMAIL:
         title: "Warte auf eine Bestätigung per E-Mail"
         subtitle: "Es war nicht möglich, das Ergebnis der Zahlung zu überprüfen. Wenn sie erfolgreich war, wirst du in Kürze eine E-Mail erhalten. Sollte diese nicht ankommen, kontaktiere bitte den Support."
+        subtitle: "Du erhältst das Ergebnis deiner Zahlung mittels E-Mail an {{email}}."
```

Comment:
Here we have a duplicated key. @dhinterlechner could you please double check it?

Reply:
forgot to delete the old one 🙈